### PR TITLE
New Cheat Sheet: Rails Console

### PR DESCRIPTION
**Pull request by GPhantom**
## Description of new Instant Answer, or changes

Reference for rails console commands.

## People to notify

@gautamkrishnar 

---

Instant Answer Page: https://duck.co/ia/view/rails_console_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/rails-console.json [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+	"Method Chaining",
+	"Deletion"
+    ],
+    "sections": {
+        "Start": [{
+            "val": "Start rails console",
+            "key": "rails console"
+        }],
+        "Create": [{
+            "val": "creates a new record with key1 equals to value1",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/rails-console.json]: Capitalise the first letter of val field

File: share/goodie/cheat_sheets/json/rails-console.json [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    },
+    "aliases": [
+        "ruby on rails console"
+    ],
+    "template_type": "terminal",
+    "section_order": [
+        "Start",
+        "Create",
+        "Read",
+	"Method Chaining",
```
Comment (gautamkrishnar) [inline share/goodie/cheat_sheets/json/rails-console.json]: Fix indentation

File: share/goodie/cheat_sheets/json/rails-console.json [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+	"Deletion"
```
Comment (gautamkrishnar) [inline share/goodie/cheat_sheets/json/rails-console.json]: here too...

**Comment by gautamkrishnar** [comment]
@jophab  Need your review....

**Comment by gautamkrishnar** [comment]
@GPhantom remember to check your JSON's indentation using some validators like JSONlint

**Comment by GPhantom** [comment]
@gautamkrishnar Valid json :+1:

**Comment by daxtheduck** [comment]
@GPhantom this is now merged! It should be live on DuckDuckGo within one week (or during the next release) and we'll be sure to notify you when it happens.

After your Instant Answer goes live for the World to use, you will be able to see [traffic information](http://docs.duckduckhack.com/maintaining/tools.html#monitor-your-instant-answer-page) about your IA as well as receive feedback directly from users. 
                
Please refer to the [maintenance guide and docs](http://docs.duckduckhack.com/maintaining/guidelines.html) to keep up with potential changes to your Instant Answer.
                
Thank you again for your help!

**Pull request merged**